opencti-platform/opencti-front/src/components/SelectField.jsx

```diff
@@ -79,7 +81,8 @@ const SelectField = (props) => {
         {props.label}
       </InputLabel>
       <MuiSelect
-        {...fieldToSelect(props)}
+        {...otherProps}
+        value={value ?? ''}
```

Comment:
Reason: `value` can be null. But an input should not be initialized with null or undefined value, instead use an empty string. By so, React can properly track changes with the input.